.github/workflows/doc.yml

```diff
@@ -14,13 +14,24 @@ jobs:
       - name: Build static files
         env:
           CARGO_TERM_COLOR: always
-        id: build
         run: |
           cd charon
           make doc
 
+      - name: Install rustc internal docs
+        run: |
+          rustup toolchain list -v
+          # Find toolchain path
+          export TOOLCHAIN=$(rustup toolchain list -v | grep active | awk '{print $NF}')
```

Comment:
That command doesn't work on my machine. I think `$(rustc --print sysroot)/share/doc/...` would more more robustly.

Reply:
It also doesn't work on my machine, because the pattern string for grep is `override` if multiple toolchains are installed...

I changed that due to github runner behavior.



> I think `$(rustc --print sysroot)/share/doc/...` would more more robustly.

Good idea. I'll update it.